test/regression/test_deploy.py

```diff
@@ -10,12 +10,17 @@
 from utils import Logger, StopWatch
 
 
+def get_expected_cortx_control_pods(namespace):
+    cmd = f'helm get values cortx --all --namespace {namespace} --output yaml | yq .control.replicaCount'
+    child = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE)  # nosec
+    return int(child.communicate()[0])
```

Comment:
FYI, this raises an Exception if the output is "null" for some reason. Probably a contrived example, but I ran this manually when the chart wasn't installed, which will reproduce it.